test/built-ins/BigInt/parseInt/S18.2.5-2-1.js

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2012 Ecma International.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-bigint-parseint-string-radix
+description: >
+    parseInt - 'S' is the empty string when inputString does not
+    contain any such characters
```

Comment:
so many things to start:

- please fix the file names, don't use the legacy format.
- please don't reuse legacy descriptions
- the info tag is missing or broken in tests. It needs to contain the respective spec text or other information that won't fit in the description line.
- the copyright headers are not valid
- missing the features flags with the valid features.
- If you use arrow functions, include it into the features flags as well